# Make use of EasyBuild environmental variables
# SEP 1st 2020 by B. Hajgato (UGent)
--- Makefile.org	2018-02-02 02:41:30.000000000 +0100
+++ Makefile	2020-09-01 13:12:05.438782738 +0200
@@ -13,11 +13,11 @@
   SONAME=install_name
 endif
 #CFLAGS=-Wall -Wno-unused-function -g -ansi -fPIC -pthread
-CFLAGS=-Wall -Wno-unused-function -O3 -ansi -fPIC -pthread
+CFLAGS?=-Wall -Wno-unused-function -O3 -ansi -fPIC -pthread
 LIB_TAG=0.3.0
-CC=gcc
-PREFIX=/usr
-LIBDIR=$(PREFIX)/lib
+CC?=gcc
+#PREFIX=/usr
+#LIBDIR=$(PREFIX)/lib
 SRC=sonic.c
 FFTLIB=
 ifeq ($(USE_SPECTROGRAM), 1)
